### ISSUE TITLE
Clash with xfce4-screenshooter causing shadowed screenshots on selected region

### ISSUE THREAD

**Issue by bunnybooboo** [chjj/compton#338]
I'm running BunsenLabs Linux and there seems to be a recognised issue with Compton causing greyed out screencap when running 'select a region'. Mentioned [HERE](https://forums.bunsenlabs.org/viewtopic.php?id=250)
The suggested fix `scrot -s` does not seem to work for me, even via terminal. If I disable Compton, however, it works as expected.

Hoping a workaround may be relatively simple and submitting in here for any suggestions.

**Comment by theferrit32** [comment]
Can confirm this issue exists on Arch linux as well, with xfce4-screenshooter.  This does not happen with the gnome-screenshot utility.

**Comment by scottmmjackson** [comment]
BunsenLabs, same issue. Reported to xfce4-screenshooter bug tracker as well.

**Comment by ryukinix** [comment]
Yes, I can confirm that same issue on my machine with XFCE@Manjaro and using xfce4-screenshooter.

**Comment by ryukinix** [comment]
A quick fix for that is ignore the shadowzing effect for xfce4-screenshooter on `~/.config/compton.conf`:

```
shadow-exclude = [              # Exclude conditions for shadows.
 "class_g ?= 'Xfce4-screenshooter'"
];
```

**Comment by pawamoy** [comment]
Instead of `class_g ?=` I used `i:e:`. Example from the link given above:

```
shadow-exclude = [ "i:e:Conky", "i:e:xfce4-screenshooter" ];
```

Shadow will still appear while taking the screenshot, but the saved image will be good.

**Comment by ryukinix** [comment]
Nice! What `i:e:` stands for?

**Comment by pawamoy** [comment]
I guess it's "ignore-case" and "exact"
